types/serverless/index.d.ts

```diff
@@ -77,7 +78,7 @@ declare class Serverless {
     getVersion(): string;
 
     cli: {
-        log(message: string): null;
+        log(message: string, entity?: string, options?: { color?: string, bold?: boolean, underline?: boolean, entity?: string }): null;
```

Comment:
Given its size, I think it would be best to pull this out into a `LogOptions` interface

Reply:
Great idea 👍🏼 I've added the `LogOptions` interface as you suggested [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54264/commits/8eb67cb96c00850245395ea30adfa2a913682948)